src/main/java/com/ace/alfox/game/MoveAction.java

```diff
@@ -21,18 +21,18 @@
     public MoveAction() {}
 
     @Override
-    public Result applyAction(GameCharacter player, Map<String, Object> params) {
-        Result result = new Result(player);
+    public ActionWebResult applyAction(Player player, Map<String, Object> params) {
+        ActionWebResult result = new ActionWebResult(player);
         String direction = (String) params.get("direction");
 
         if(player.hp == 0)
             return result.notOk().log("You're dead.");
         if(direction == null || !directions.containsKey(direction))
-            return result.notOk().log("Unknown direction"); // notice .log returns `Result`
+            return result.notOk().log("Unknown direction"); // notice .log returns `ActionWebResult`
 
         // player.attacking.clear() // if they were attacking anyone, they aren't anymore
                                     // maybe do running away mechanic here?
-        player.location = player.location.mathAdd(directions.get(direction));
+        player.location = player.location.add(directions.get(direction));

```

Comment:
We don't have to do this for this PR, but we should add rate limiting so that someone can't spam 100k requests and move super fast. Lots of approaches we could take, but let's chat out of the PR about it.

^Not really an MVP feature, so fine to skip.